packages/widget/src/components/internal/dropdown/dropdown.ts

```diff
@@ -0,0 +1,128 @@
+import { html, type HTMLTemplateResult } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { map } from 'lit/directives/map.js';
+import { when } from 'lit/directives/when.js';
+import { chevronIcon, networkIconsMap } from '../../../assets';
+import { capitalize } from '../../../utils';
+import { BaseComponent } from '../../base-component/base-component';
+import { styles } from './styles';
+
+export interface DropdownOption {
+  id?: string;
+  name: string;
+  icon?: HTMLTemplateResult | string;
+}
+
+@customElement('dropdown-component')
+export class Dropdown extends BaseComponent {
+  static styles = styles;
+
+  @state()
+  _isDropdownOpen = false;
```

Comment:
same question here as below, since this is a internal state variable, we could just renamed as `isDropdownOpen`